/**:
  ros__parameters:
    control_command: "basic"
    # Which index in the joy message corresponds to which axis
    axes:
      # In our version of the joystick driver, axes typically go from -1.0 to 1.0
      left_cmd: 1
      right_cmd: 3
    # Scale and offset of axes: val = (raw * scale) + offset
    axis_scale:
      left_cmd: 127.0
      right_cmd: 127.0
    axis_offset:
      left_cmd: 0.0
      right_cmd: 0.0
    # Buttons are not yet implemented
    # Index for which button is which
